README.md

```diff
@@ -1,8 +1,8 @@
 GRNsight
 ========
 [![DOI](https://zenodo.org/badge/16195791.svg)](https://zenodo.org/badge/latestdoi/16195791)
-[![Build Status](https://travis-ci.org/dondi/GRNsight.svg?branch=master)](https://travis-ci.org/dondi/GRNsight)
-[![Coverage Status](https://coveralls.io/repos/github/dondi/GRNsight/badge.svg?branch=master)](https://coveralls.io/github/dondi/GRNsight?branch=master)
+[![Build Status](https://travis-ci.org/dondi/GRNsight.svg?branch=beta)](https://travis-ci.org/dondi/GRNsight)
+[![Coverage Status](https://coveralls.io/repos/github/dondi/GRNsight/badge.svg?branch=beta)](https://coveralls.io/github/dondi/GRNsight?branch=beta)
```

Comment:
These look like they shouldn't change when going to a merge w/ `master`.

Reply:
Yes, I was planning on doing it as a hotfix upon merge